mpt/impl/imap-mailbox/core/src/main/resources/org/apache/james/imap/scripts/SearchAtoms.test

```diff
@@ -1716,15 +1713,9 @@ S: A82 OK SEARCH completed\.
 C: A83 SEARCH 17:*
 S: \* SEARCH 17 18 19
 S: A83 OK SEARCH completed\.
-C: A84 SEARCH OLD
-S: \* SEARCH
-S: A84 OK SEARCH completed\.
 C: A85 SEARCH OR ANSWERED FLAGGED
 S: \* SEARCH 1 2 3 4 5 6 7 8 9 10 11 12 13 14
 S: A85 OK SEARCH completed\.
-C: A86 SEARCH RECENT
```

Comment:
isn't it worth to replace these 3 IMAP calls by there actual results?

Reply:
They return nothing, that's a commit to contnue the build process, but I already know if we want to move further on this topic that we might need a rework of RECENT handling in the IMAP layer.